source/agora/consensus/ValidatorSet.d

```diff
@@ -160,6 +160,25 @@ public class ValidatorSet
         }
     }
 
+    /***************************************************************************
+
+        Remove all validators from the validator set
+
+    ***************************************************************************/
+
+    public void removeAll () @trusted nothrow
+    {
+        try
+        {
+            // SQLite has added a TRUNCATE optimizer to the DELETE statement.
```

Comment:
What is this comment for? Is it saying we should use TRUNCATE? Or not? It's only confusing, maybe we can remove it..

Reply:
Most RDBMS uses `TRUNCATE` in this case.
SQLite is information that is optimized for delete without supporting TRUNCATE.

Reply:
Then this comment is unnecessary, it's an optimization detail that has nothing to do with Agora. Please remove it. :)